packages/calcite-components/src/components/card/card.scss

```diff
@@ -1,87 +1,114 @@
+@import "~@esri/calcite-design-tokens/dist/scss/core";
+@import "~@esri/calcite-design-tokens/dist/scss/global";
+
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-card-background-color: Specifies the background color of the component.
+ * @prop --calcite-card-box-shadow: Specifies the box shadow of the component.
+ * @prop --calcite-card-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-card-subtitle-color: Specifies the color of the component's `"subtitle"` slot.
+ * @prop --calcite-card-title-color: Specifies the color of the component's `"title"` slot.
+ *
+ */
+
+/*
+  * Internal Properties
+  *
+  * These properties are intended for internal component use only. It is not recommended that these properties be overwritten.
+  *
+  * --calcite-internal-card-background-color
+  * --calcite-internal-card-border-color
+  * --calcite-internal-card-box-shadow
+  * --calcite-internal-card-corner-radius
+  * --calcite-internal-card-subtitle-color
+  * --calcite-internal-card-title-color
+*/
+
 :host {
   @apply block max-w-full;
-  & .calcite-card-container {
-    @apply bg-foreground-1
-    border-color-2
-    text-color-3
-    relative
+  --calcite-internal-card-background-color: var(--calcite-card-background-color, var(--calcite-color-foreground-1));
```

Comment:
I don't think we need to duplicate CSS vars for each token just to have an "-internal-". Just apply the component token directly.

Reply:
Removed.